server/controllers/journal.js

```diff
@@ -238,6 +235,17 @@ function handleSales (id, user_id, done, caution) {
     'WHERE `sale`.`uuid`=' + sanitize.escape(id) + ' ' +
     'ORDER BY `sale_item`.`credit`;';
 
+
+  function getSubsisy () {
+    var sql =
+    'SELECT `sale_subsidy`.`value`, `debitor_group`.`account_id`, `subsidy`.`text`, `sale`.`uuid` FROM ' +
+    '`sale_subsidy`, `debitor_group`, `subsidy`, `sale` WHERE `sale_subsidy`.`sale_uuid`=`sale`.`uuid` AND ' +
+    '`sale_subsidy`.`subsidy_uuid`=`subsidy`.`uuid` AND `subsidy`.`debitor_group_uuid`=`debitor_group`.`uuid` AND ' +
+    '`sale_subsidy`.`sale_uuid`=' + sanitize.escape(id);
+
+    return db.exec(sql);
```

Comment:
You should use db.exec()'s built-in escapes.  Instead of `sale_uuid=' +sanitize.escape(id)`, you can write `sale_uuid=?;'` and then pass the id to `db.exec(sql, [id])`.